CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Add child client support for AwsSigV4 ([#725](https://github.com/opensearch-project/opensearch-js/pull/725))
```

Comment:
I would say something simpler, e.g. "Inherit AwsSigV4 in .child" or something like that.

Reply:
Got it. Updated changelog.